### DISCUSSION TITLE
https://notes.alexkehayias.com/emacs-completion-stack/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#10] [Announcements]
# https://notes.alexkehayias.com/emacs-completion-stack/

There are many packages for completions in Emacs. I recently switched over from ido and helm to the MOVEC stack: Marginalia, Orderless, Vertico, Embark, Consult.

https://notes.alexkehayias.com/emacs-completion-stack/

**Comment by jdkschang** [comment]
is there a public branch representative of these changes? asking for a friend 😎

**Reply by alexkehayias** [reply]
@jdkschang You can find the work in progress diff here. 

https://github.com/alexkehayias/emacs.d/compare/ak/consult-vertico-embark?expand=1

Roughly: remove `ido` and `helm` from everywhere, add the new completion stack, then add back in some customized wrappers around `org-ql` and `org-roam`. 

So far, everything is working for me!
